### PR TITLE
Setup CI

### PR DESCRIPTION
**Pull request by rock3r**
## Problem

We don't have a CI for this project

## Solution

Setup a CI with CircleCI that tries building the Android debug APK and runs `flutter analyze`. I haven't found an easy way to setup a macOS CI for the project to build the iOS version, maybe @niamh-power could help with that.

Next in my todo list is to make sure there is no formatting violations using `dartfmt -n` (dry run) and assess it outputs an empty list of issues. Also pending is sharing the build state as much as possible in the CI workflow to speed things up (no need to always run flutter doctor?)

### Test(s) added

No.

### Paired with 

Nobody

### CODE REVIEW

File: .circleci/ci-scripts/ci-mock-google-services-setup.sh [hunk 1]
```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
```
Comment (rock3r) [inline .circleci/ci-scripts/ci-mock-google-services-setup.sh:1]: This file is a copy of [this one](https://github.com/squanchy-dev/squanchy-android/blob/develop/.circleci/ci-scripts/ci-mock-google-services-setup.sh) but adapted to also output a dummy plist for the iOS app

File: .circleci/ci-scripts/ensure-android-sdkmanager.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```
Comment (rock3r) [inline .circleci/ci-scripts/ensure-android-sdkmanager.sh:1]: This file is a copy of [this](https://github.com/squanchy-dev/squanchy-android/blob/develop/.circleci/ci-scripts/ci-ensure-android-sdkmanager.sh), not sure if we'll need something like this for iOS

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -12,10 +12,10 @@ _Explain what you did test and what you didn't, and why_
 
 ### Screenshots
 
-| Before | After |
-| ------ | ----- |
-| gif/png _before_ | gif/png _after_ |
+ Before | After
```
Comment (rock3r) [inline .github/pull_request_template.md:15]: I took the opportunity to clean this up a bit as I needed file changes to trigger the CI for 💃

File: lib/utils/button_widgets.dart [hunk 4]
```diff
@@ -1,5 +1,3 @@
-import 'dart:async';
```
Comment (rock3r) [inline lib/utils/button_widgets.dart:1]: This was flagged by the `flutter analyze` as unused

**Review by xrigau — APPROVED** [review]
LGTM 👍

**Comment by rock3r** [comment]
It's approved, imma shipit myself so we can merge it in #55

**Pull request merged**